lib/cli.js

```diff
@@ -139,7 +139,7 @@ cli.generateParserString = function generateParserString(opts, grammar) {
     opts = opts || {};
     var jison = require('./jison.js');
 
-    var settings = grammar.options || {};
+    var settings = grammar.options || opts;
```

Comment:
Should this really check either or, or would it desirable to merge `grammar.options` and `opts` with e.g. [extend](https://www.npmjs.com/package/extend)?